src/Sentry/Platforms/Android/SentrySdk.cs

```diff
@@ -59,6 +59,11 @@ private static void InitSentryAndroidSdk(SentryOptions options)
 
         // Define the configuration for the Android SDK
         SentryAndroidOptions? nativeOptions = null;
+
+        // Convert NdkHandlerStrategy .net to Java Android
```

Comment:
nit: inconsistent. You're capitalizing Java but not .NET
Either go with Java and .NET or, java and .net
```suggestion
        // Convert the .NET NdkHandlerStrategy to Java / Android
```